contributing/changelog-process.md

```diff
@@ -151,3 +151,8 @@ A PR check is included to validate the changelog entry file.
 If a PR doesn't need a changelog entry its check can be skipped:
 - Adding the label `skip-changelog-check` to the PR.
 - Check in PRs with title `chore`, `test`, `doc` or `ci` is automatically skipped. However a changelog can still be added if needed.
+
+## Unreleased section of CHANGELOG.md automatic update
```

Comment:
We can add this title in the index at the top of the file so it has visibility.

Reply:
Done!